partitioned/src/main/java/org/apache/geode/examples/partitioned/BaseClient.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Nitpick - I feel like this BaseClient actually makes the code less readable. It might be better to just create the caches and regions inline in the producer and consumer, and just have a static class that has the common properties.

Also, getRegion1 and getRegion2 aren't very descriptive method names.